presto-docs/src/main/sphinx/connector/clp.rst

```diff
@@ -94,6 +96,103 @@ If you prefer to use a different source--or the same source with a custom implem
 implementations of the ``ClpMetadataProvider`` and ``ClpSplitProvider`` interfaces, and configure the connector
 accordingly.
 
+Metadata Filter Config File
+----------------------------
+
+The configuration file defines metadata filters for different scopes:
+
+- **Catalog-level**: applies to all schemas and tables under the catalog.
+- **Schema-level**: applies to all tables under the specified catalog and schema.
+- **Table-level**: applies to the fully qualified ``catalog.schema.table``.
+
+.. note::
```

Comment:
Personally, I don't think we need these many note blocks, not normal paragraphs or sentences would be fine. What do you think @kirkrodrigues